.coveragerc

```diff
@@ -4,7 +4,12 @@
 omit =
     */test*
     */__init__.py
-source = qiita_core,qiita_ware,qiita_db,qiita_pet 
+
+    # these files will be ommited from coveralls because they contain test
+    # of the environment and methods they are executed regularly by travis,
+    # then there should not be concern over their "correctness".
+    qiita_core/environment_manager.py
+source = qiita_core,qiita_ware,qiita_db,qiita_pet
```

Comment:
Yeah, that makes sense, or keeping #120 open and in this milestone. If
we can't add tests for the CLIs, it might also be worth to just run all
the scripts with the --help option and make sure none of them error
in the travis build, I think that would be a *baby step* :P

On (Feb-13-15|10:00), Joshua Shorenstein wrote:
> > @@ -4,7 +4,12 @@
> >  omit =
> >      */test*
> >      */__init__.py
> > -source = qiita_core,qiita_ware,qiita_db,qiita_pet 
> > +
> > +    # these files will be ommited from coveralls because they contain test
> > +    # of the environment and methods they are executed regularly by travis,
> > +    # then there should not be concern over their "correctness".
> > +    qiita_core/environment_manager.py
> > +source = qiita_core,qiita_ware,qiita_db,qiita_pet
> 
> Is more for whether we want to add `scripts` to source or not. If it's in there now with this pull request, it will lower the score and we'll remember to add those tests.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/qiita/pull/853/files#r24682857